### ISSUE TITLE
Test features added by super-class are not counted in summary report

### ISSUE THREAD

**Issue by renatoathaydes** [renatoathaydes/spock-reports#174]
Given a class:

```groovy
import spock.lang.Specification

class BaseTestClass extends Specification {

    def "Always true"() {
        expect: 'true is true'
        true
    }

}
```

And a sub-class:

```groovy
class SubclassTest extends BaseTestClass {

    def "Also always true"() {
        expect: 'always true also'
        true
    }
}
```

The summary report shows:

| Name | Features | Failed | Errors | Skipped | Success rate | Time |
|----------|-------------|----------|----------|------------|-------------------|---------|
| com.athaydes.SubclassTest | 1 | 0 | 0 | 0 | 100.0% | 0.034 seconds |

It shows only 1 feature instead of 2, as it should have.

**Comment by renatoathaydes** [comment]
Problem found while checking https://github.com/renatoathaydes/spock-reports/issues/144

**Comment by renatoathaydes** [comment]
Fixed in version 1.6.3.
